src/components/home/index.tsx

```diff
@@ -1,24 +1,21 @@
 import * as React from 'react'
 import {FC} from 'react'
-import Slider from './slider/Slider'
-import Center from './center/Center'
-import WelcomeSection from './welcomeSection/WelcomeSection'
-import Footer from './footer/Footer'
-import {Movie} from './slider/SliderModal'
+import {Center} from './center/Center'
+import {WelcomeSection} from './welcomeSection/WelcomeSection'
+import {Footer} from './footer/Footer'
+import {SliderComponent} from './slider/slider'
 
-const Home: FC<{movies: Movie[]; loading: boolean; id: number}> = ({
+export const Home: FC<{movies: Movie[]; loading: boolean; id: number}> = ({
   movies,
   loading,
   id,
 }) => {
   return (
     <>
-      <Slider movies={movies} loading={loading} />
+      {loading ? null : <SliderComponent movies={movies} />}
```

Comment:
```suggestion
      {!loading && <SliderComponent movies={movies} />}
```

It's more or less the same thing with just fewer characters

Reply:
Yeah looks more neat! I changed it.